modules/raft/src/main/java/org/apache/ignite/raft/jraft/Iterator.java

```diff
@@ -33,7 +33,7 @@ public interface Iterator extends java.util.Iterator<ByteBuffer> {
     /**
      * Return a unique and monotonically increasing identifier of the current task: - Uniqueness guarantees that
      * committed tasks in different peers with the same index are always the same and kept unchanged. - Monotonicity
-     * guarantees that for any index pair i, j (i < j), task at index |i| must be applied before task at index |j| in
+     * guarantees that for any index pair i, j (i less than j), task at index |i| must be applied before task at index |j| in
```

Comment:
Please rollback changes in the `jraft` package and exclude it from the javadoc tool

Reply:
Do you mean we should not generate javadoc for jraft package?